src/routes/App/style.css

```
@@ -1,6 +1,14 @@
-@font-face {
-  font-family: 'VAGRounded';
-  src: url('../../components/assets/fonts/vagrounded-thin.ttf') format('truetype');
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,700&display=swap');
+@import url('https://fonts.googleapis.com/css?family=Poppins:400,500,700&display=swap');
```

you can import both fonts in the same URL 
---
```suggestion
@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700|Poppins:300,400,500,700&display=swap');
```